### ISSUE TITLE
Unable to apply an event-provided badge

### ISSUE THREAD

**Issue by chadberg** [neinteractiveliterature/intercode#6945]
I am attempting to convert https://u.interconlarp.org/user_con_profiles/139329 to an event-provided badge for "Pilgrimage", but it is not listed as an option in the dropdown menu when asked to select the event to associate with. No items are in that menu. I am using Firefox 105.0.3, and have logged out and back in. I have disabled adblocker and enhanced blocking.

**Comment by nbudin** [comment]
Ah yeah, sorry about this - it's a little counterintuitive, but you have to start typing the name of the event.  I just tried this while "become user"-ed to you, and it does appear to work when I do that.

(I'd love any ideas for ways to make this easier to figure out!)

**Comment by chadberg** [comment]
Ah! Must be related to the 'search as you type'. No way to preload the list of games the person is already a GM for?

**Comment by chadberg** [comment]
Maybe instead of the hint text saying 'Select event' it should say 'Search for event' ?

**Comment by chadberg** [comment]
Separate concern, but with the same failure concern. Still unable to convert this badge to an event provided badge, getting the error message "No such charge: 'ch_1GKc882kmnb9nvAiE8VsHKKW'" - I am reading this as the API not being able to go back 2.5+ years to when the original badge was purchased. If that is the case, I don't think a technical solution is warranted, given the rarity of any convention being delayed three years. I will instead manually refund the purchase in stripe and delete the existing badge, creating a new one associated with the comp.

**Comment by nbudin** [comment]
Yeah, I think that's exactly it.  This shouldn't normally happen but nothing about Intercon U is normal.
